igor-web/src/test/groovy/com/netflix/spinnaker/igor/travis/client/model/ConfigSpec.groovy

```diff
@@ -73,4 +74,21 @@ class ConfigSpec extends Specification {
         genericParameterDefinitionList.first().defaultValue == '"foo=bar"'
     }
 
+    @Unroll
+    def "getSecrets returns secrets"(){
+        given:
+        Config config = new Config([:])
+        config.globalEnv = globalEnv
+
+        expect:
+        config.getSecrets() == secrets
+
+        where:
+        globalEnv                          || secrets
+        [ 'FOO=bar' ]                      || []
+        [ 'FOO=bar', ["secure":"xyz"] ]    || [ [secure:"xyz"] ]
+        [ [secure:"xyz"], [secure:"abc"] ] || [ [secure:"xyz"], [secure:"abc"] ]
```

Comment:
I have no idea why secrets are represented this way and I don't know why we have to send them when triggering. If we do not collect the secrets and send them, the triggered build will not have these secrets. I have reached out to travis on this and I hope they will fix it so that we do not need to send the secrets and can remove that part of the code.